dev.neovide.neovide.yml

```diff
@@ -0,0 +1,51 @@
+id: dev.neovide.neovide
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+# sdk-extensions:
+#   - org.freedesktop.Sdk.Extension.rust-stable
+command: neovide
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri # opengl
+  - --share=network
+  - --share=ipc
+  - --filesystem=host
+  - --filesystem=xdg-config/neovide
+  - --talk-name=org.freedesktop.Flatpak
+modules:
+  - name: neovide
+    sources:
+      # - type: archive
+      #   url: https://github.com/neovide/neovide/archive/refs/tags/0.12.2.tar.gz
+      #   sha256: 5425c60454388651fd79757bde7c4d7499cdc49b375f7697b48d8366d45d08e4
+      # - generated-sources.json
+      - neovide-git.json
+      - type: file
+        url: https://github.com/sqwxl/neovide/releases/download/0.12.3/neovide-linux-x86_64.tar.gz
```

Comment:
how does this work for aarch64?

Reply:
There is no ARM build for neovide: https://github.com/neovide/neovide/releases

Reply:
so how does that work in the aarch64 flatpak?

Reply:
can't you build from source instead?

Reply:
I agree that it would be better to build from source, but there's an upstream bug with that was preventing flatpak-builder from completing the compilation with cargo. (I left the code for this as commented-out lines in the manifest). Once the upstream bug is resolved, I'll uncomment those and remove the reference to a pre-built binary.

Reply:
Added a `flathub.json` specifying supported arch